dev/DynamicDependency/M.AM.DD.AddPackageDependencyOptions.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+
+#include "M.AM.DD.AddPackageDependencyOptions.h"
```

Comment:
I find this file naming pattern confusing and unconventional, and would prefer the full name for searching by filename to work as expected.

Reply:
`Microsoft.ApplicationModel.DynamicDependencies.Whatever.blah` is a long name that works poorly with Visual Studio's Solution Explorer, VSCode's file list and other tools.

Not critical to block PR so I've added a note to consider changing this in [MSIX Dynamic Dependencies PR follow ups #403](https://github.com/microsoft/ProjectReunion/issues/403) in a future PR.